aws/resource_aws_glue_crawler_test.go

```diff
@@ -1391,6 +1516,146 @@ resource "aws_glue_crawler" "test" {
 `, rName, rName, path1, path2)
 }
 
+func testAccGlueCrawlerConfig_CatalogTarget_TableResources(tables []string) string {
+	var config string
+	for _, table := range tables {
+		config += fmt.Sprintf(`
+      resource "aws_glue_catalog_table" %q {
+        database_name = "${aws_glue_catalog_database.test.name}"
+        name          = %q
+        table_type    = "EXTERNAL_TABLE"
+
+        storage_descriptor {
+          location      = "s3://${aws_s3_bucket.default.bucket}"
+        }
+      }
+    `, table, table)
+	}
+
+	return config
+}
+
+func testAccGlueCrawlerConfig_CatalogTarget_TableList(tables []string) string {
```

Comment:
Using the noted reference above (`aws_glue_catalog_table.test[*].name`), this function is extraneous. 👍